recipes/examples/python_recipe.py

```diff
@@ -1,128 +0,0 @@
-#!/bin/python3
```

Comment:
This is a valid python recipe I think, we should probably just move this into the `lnst/Recipes/...` hierarchy

Reply:
resolved with commit https://github.com/LNST-project/lnst/pull/205/commits/658427390b75698fdc47080154bcbf6ff53f4a19